.gitignore

```diff
@@ -0,0 +1,4 @@
+/venv/
+/.ipynb_checkpoints/
+/prefetch.log
```

Comment:
I took out prefetch log in the last refactor but I will probably add it back in so leaving this makes sense for now to me :)

Reply:
Heh, happy to take it out as well...